tests/testsuite.src/syn_file.at

```diff
@@ -3134,7 +3135,7 @@ AT_CHECK([$DIFF reference tspfilex.ddl], [0], [], [])
 AT_CAPTURE_FILE(./tspfilex.xd)
```

Comment:
while this is unrelated, maybe you can fix that line in this commit as well - the .xd file is expected to be there even if the ddl file check did not work, so it should be moved directly after capture of the ddl (if you do that, it is enough to do it on upstreaming)